app/code/Magento/Catalog/etc/db_schema.xml

```diff
@@ -573,6 +577,40 @@
             <column name="store_id"/>
         </index>
     </table>
+    <table name="catalog_compare_list" resource="default" engine="innodb" comment="Catalog Compare List with hash Table">
+        <column xsi:type="int" name="list_id" padding="10" unsigned="true" nullable="false"
+                identity="true" comment="Compare List ID"/>
+        <column xsi:type="int" name="customer_id" padding="10" unsigned="true" nullable="true" identity="false"
+                comment="Customer ID"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="list_id"/>
+        </constraint>
+        <constraint xsi:type="foreign" referenceId="CATALOG_COMPARE_LIST_CUSTOMER_ID_CUSTOMER_ENTITY_ENTITY_ID"
+                    table="catalog_compare_list" column="customer_id" referenceTable="customer_entity"
+                    referenceColumn="entity_id" onDelete="CASCADE"/>
+        <index referenceId="CATALOG_COMPARE_LIST_VISITOR_ID_CUSTOMER_ID" indexType="btree">
+            <column name="customer_id"/>
+        </index>
+    </table>
+    <table name="list_id_mask" resource="default" engine="innodb" comment="List ID and masked ID mapping">
```

Comment:
@Usik2203 why do we need a separate table for this. A new column on catalog_compare_list should suffice right?

Reply:
@prabhuram93  
I followed example of `createEmptyCart` mutation during creation `createCompareList` mutation

 It seems me that logic of `createEmptyCart` is very similar with `createCompareList` .

`createEmptyCart` creates and stores hash of quotes in` quote_id_mask` table

I thought that it's good example and created `list_id_mask` table for storing hash of compare lists

Thanks

Reply:
@Usik2203 It was performed that way in quote, because id masking was an additive feature to quote and "quote" table existed before that feature. Since we are introducing "catalog_compare_list" in this pr, it makes sense to add one more column for the mask, instead of having a new table.

Reply:
I will do this change.
Thanks

Reply:
**Done** Table was removed